vello_encoding/src/encoding.rs

```diff
@@ -454,7 +451,8 @@ impl Encoding {
             .extend_from_slice(bytemuck::cast_slice(bytemuck::bytes_of(&DrawImage {
                 xy: 0,
                 width_height: (brush.image.width << 16) | (brush.image.height & 0xFFFF),
-                sample_alpha: ((brush.image.format as u32) << 14
+                sample_alpha: ((brush.image.alpha_type as u32) << 15
```

Comment:
This seems like a strange ordering; why encode the format in a constrained area? We know that the `AlphaType` is always just going to be a single bit, whereas it seems reasonable that we'd gain more image formats.

Reply:
Indeed, I will fix this.